tests/func/test_get_url.py

```diff
@@ -13,6 +13,16 @@ def test_get_file(tmp_dir):
     assert (tmp_dir / "foo_imported").read_text() == "foo contents"
 
 
+def test_get_dir(tmp_dir):
```

Comment:
Should we mark that the purpose of this test is to check no-cache use case?

Reply:
Pwerl,
Before we go to the tests, could you please confirm that fix is done in correct place and in corret manner.

Answering your question:
> Should we mark that the purpose of this test is to check no-cache use case?

This test-suit targets high-level function, get_url, on this level there is no concept of cache, it goes only on lower levels, so we should not mention cache here at all.

Also it is probably needed to have another test for no repo/cache case in test_repo, test_run_cache, test_dvc suits where we could go into "nocache" details.
Let me know what you think.